package.json

```diff
@@ -37,24 +42,35 @@
 		"cmd",
 		"console"
 	],
-	"dependencies": {
+	"dependencies": {},
+	"devDependencies": {
+		"@babel/core": "^7.1.0",
+		"@babel/preset-env": "^7.1.0",
+		"@babel/register": "^7.0.0",
+		"ava": "^1.0.0-beta.8",
+		"babel-core": "^7.0.0-bridge.0",
 		"camelcase-keys": "^4.0.0",
 		"decamelize-keys": "^1.0.0",
+		"esm": "^3.0.84",
+		"execa": "^0.10.0",
+		"indent-string": "^3.0.0",
 		"loud-rejection": "^1.0.0",
 		"minimist-options": "^3.0.1",
 		"normalize-package-data": "^2.3.4",
 		"read-pkg-up": "^3.0.0",
 		"redent": "^2.0.0",
+		"rollup": "^0.66.2",
+		"rollup-config": "^1.0.1",
+		"rollup-plugin-commonjs": "^9.1.8",
+		"rollup-plugin-json": "^3.1.0",
+		"rollup-plugin-license": "^0.7.0",
+		"rollup-plugin-node-resolve": "^3.4.0",
 		"trim-newlines": "^2.0.0",
+		"xo": "*",
 		"yargs-parser": "^10.0.0"
 	},
-	"devDependencies": {
-		"ava": "*",
-		"execa": "^0.10.0",
-		"indent-string": "^3.0.0",
-		"xo": "*"
-	},
 	"xo": {
+		"semicolon": false,
```

Comment:
Don't do unrelated changes. Same applies to all the removing of semicolons. You might not like them, but you should follow what the project already uses.